docs/content/docs/vagrant__introduction.md

```diff
@@ -16,6 +16,8 @@ $ git clone https://github.com/Kitware/HPCCloud-deploy.git
 $ git clone https://github.com/Kitware/cumulus.git
 $ git clone https://github.com/Kitware/girder.git
 
+$ ln -s HPCCloud hpccloud
```

Comment:
Lets just change the clone command above to be 

```
git clone https://github.com/Kitware/HPCCloud.git hpccloud

```

Then we don't have to add this extra step?